golem-base-indexer/golem-base-indexer-logic/src/repository/timeseries/block_transactions.rs

```diff
@@ -0,0 +1,74 @@
+use anyhow::{Context, Result};
+use sea_orm::{prelude::*, DbBackend, FromQueryResult, Statement};
+use sea_query::{Expr, Iden, PostgresQueryBuilder, Query, SelectStatement};
+use tracing::instrument;
+
+use crate::types::{BlockTransactionPoint, ChartInfo};
+
+#[derive(Iden)]
+pub enum Transactions {
+    Table,
+    BlockNumber,
+}
```

Comment:
This is a pattern for views and materialized views, as sea-orm doesn't support them natively.
In this case, use `golem_base_indexer_entity::transactions::Entity` instead